components/docs-chef-io/content/automate/ha_on_premises_deployment_prerequisites.md

```diff
@@ -75,33 +75,22 @@ Current Automate HA integrates with the following non-Chef tools:
 
 ## Hardware Requirements
 
-{{< note >}} Use a [Hardware Calculator](/calculator/automate_ha_hardware_calculator.xlsx) to check how much hardware you will need for your use case. {{< /note >}}
-
-We have some sample values based on the performance benchmarking tests to give you an apt hardware configuration. Refer to the table below to populate things in the **Hardware Calculator** according to your requirement. The table below shows the tested **assumptions** and should not be used as exact recommendations.
+{{< note >}}
 
-You can use the below assumptions in the calculator to drive into your hardware requirement:
+- Please refer to [Performance Benchmarks](/automate/ha_performance_benchmarks) for more details on the hardware requirements.
```

Comment:
Replace it with:

- Refer to [Performance Benchmarks](/automate/ha_performance_benchmarks) for more details on the hardware requirements.

Reply:
done